pkg/util/util.go

```diff
@@ -60,3 +66,42 @@ func IntPointer(x int) *int64 {
 	xout = int64(x)
 	return &xout
 }
+
+var passwordLetters = []rune("0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+// generates a [0-9a-zA-Z] password of the specified length
+func GenPassword(length int) string {
```

Comment:
The signature kind of suggests that longer passwords are stronger but the generator is determined by a single int64 seed. crypto/rand seems like a better fit, or at least re-seed every ~12 reads.

Reply:
I'm not sure reseeding would be more secure, but I'll look at moving to crypto/rand

Reply:
moved to crypto/rand now, kept math/rand as fallback